.travis.yml

```diff
@@ -62,7 +66,7 @@ script:
 - 'if [ -n "${LINT-}" ]; then npm run lint; fi'
 - 'if [ -n "${LINT-}" ]; then npm run test:type; fi'
 - 'if [ -n "${UNIT-}" ]; then npm run test:unit; fi'
-- 'if [ -n "${COMPANION-}" ]; then npm run test:companion; fi'
+- 'if [ -n "${COMPANION-}" ] || [ -n "${COMPANION_NODE6-}" ]; then npm run test:companion; fi'
```

Comment:
Since `npm install` fails on Node 6, we could instead configure the COMPANION_NODE6 build with `nodejs: stable`, and add a line above this one:
```yaml
- 'if [ -n "${COMPANION_NODE6-}" ]; then nvm install 6.0.0 && nvm use 6.0.0; fi'
```

so that the latest nodejs will be used during installation but node 6 will be used during tests.

Reply:
fixed! Thanks for the suggestion! 👌